examples/axes_grid1/demo_axes_rgb.py

```diff
@@ -1,7 +1,7 @@
 """
-=============
-Demo Axes RGB
-=============
+=======
+RGBAxes
```

Comment:
Not sure this is all that clear either - what about RGB composite image or RGB channels or even Toolkit RGB Axes/Toolkit RGB Axes helper

Reply:
Changed to "Showing RGB channels using RGBAxes" plus description update.